### DISCUSSION TITLE
Use pnpm instead of npm

### DISCUSSION THREAD

**Discussion by cloudcompute** [directus/directus#12911] [Feature Requests]
I have been using pnpm since the last several months. It is simply superb.

- very fast
- simple and straightforward, no need to do any workarounds
- takes less space than yarn and npm

Read more about it here: [https://pnpm.io/](https://pnpm.io/)

I have seen many github repositories, including popular frameworks, switching to pnpm since the last several months.

**Comment by rijkvanzanten** [comment]
This has since been addressed
